onnx2kerastl/pooling_layers.py

```diff
@@ -68,7 +71,18 @@ def convert_maxpool(node, params, layers, lambda_func, node_name, keras_name):
             name=keras_name,
             data_format='channels_first'
         )
-
+    ceil_mode = params['ceil_mode'] if 'ceil_mode' in params else False
```

Comment:
can be replace with  `params.get('ceil_mode', False)`

Reply:
Done